From 63c29ff0adfc3893adf8bcb8934720312aaf11e8 Mon Sep 17 00:00:00 2001
From: Noel Grandin <noel@peralex.com>
Date: Mon, 20 Jul 2015 10:55:59 +0200
Subject: [PATCH] fix linux buildbot

the GTK3 version check for the GDK_IS_X11_DISPLAY macro
does not appear to work well on all the machines

Change-Id: I7020afdca1b270f89910753737c2c70b61453b34
---
 vcl/unx/gtk/window/gtksalframe.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vcl/unx/gtk/window/gtksalframe.cxx b/vcl/unx/gtk/window/gtksalframe.cxx
index 2abc94f..2cb31bc 100644
--- a/vcl/unx/gtk/window/gtksalframe.cxx
+++ b/vcl/unx/gtk/window/gtksalframe.cxx
@@ -105,8 +105,8 @@
 #define IS_WIDGET_MAPPED   GTK_WIDGET_MAPPED
 #endif
 
-#if !GTK_CHECK_VERSION(3,10,0)
 #define GDK_WINDOWING_X11
+#ifndef GDK_IS_X11_DISPLAY
 #define GDK_IS_X11_DISPLAY(foo) (true)
 #endif
 
-- 
2.1.0

